packages/admin-frontend/components/EditBook/EditTabBar.js

```diff
@@ -33,33 +33,24 @@ class RouteAwareTabBar extends React.Component<Props, State> {
     selectedTab: this.props.chapterId ? 'chapter' : 'book'
   };
 
-  handleChange = (event: Event, selectedTab: 'chapter' | 'book') =>
-    this.setState({ selectedTab });
-
-  render() {
+  handleChange = (event: Event, selectedTab: 'chapter' | 'book') => {
     const {
       router: { query }
     } = this.props;
+    this.setState({ selectedTab });
+    Router.push({ pathname: `/admin/edit/${selectedTab}`, query });
```

Comment:
Handle router with onChange on Tab instead of onClick in Tabs, because otherwise I can't get hold of the value book|chapter to change the tab indicator.